<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryView</class>
 <widget class="QWidget" name="EntryView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>717</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>aakar</family>
    <pointsize>16</pointsize>
    <italic>true</italic>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>501</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/Images/Images/page.jpg);
background-repeat: repeat-y;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>499</width>
      <height>599</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="entryEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>401</width>
       <height>581</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>aakar</family>
       <pointsize>16</pointsize>
       <italic>true</italic>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-position:center;
</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>231</width>
     <height>491</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>229</width>
      <height>489</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="moodTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>211</width>
       <height>491</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>12</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Moods Last Week:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName</string>
   </property>
  </widget>
  <widget class="QLabel" name="diaryNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DiaryName</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveEntryButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>110</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editEntryButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>190</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTopicButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>270</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Topic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMoodButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>350</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Mood</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteEntryButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>430</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBackButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>580</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
